### ISSUE TITLE
add posibility to block IP address

### ISSUE THREAD

**Issue by Mannshoch** [ukanth/afwall#939]
Because /system/etc/hosts is not the place to block IP-Address but IP-Tables is. It would be nice if  you allow to set a list of IP which should get blocked.

**Comment by ThomasCr** [comment]
Yes, I am also miss such features, to control where a app can access to.
It could be also nice to define global lists which should get blocked/allowed. And then you can tag those lists to apps.

**Comment by Timmmy001** [comment]
This function would be really useful.

**Comment by ohnonot** [comment]
I'm pretty sure this is covered in the wiki's [custom script article](https://github.com/ukanth/afwall/wiki/CustomScripts).
#931 would also seem relevant.
